src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2363,6 +2363,10 @@ private function registerCacheConfiguration(array $config, ContainerBuilder $con
             ];
         }
         foreach ($config['pools'] as $name => $pool) {
+            if (\in_array('cache.app', $pool['adapters'] ?? [], true) && $pool['tags']) {
+                trigger_deprecation('symfony/framework-bundle', '7.2', 'Using the "tags" option with the "cache.app" adapter is deprecated. You should either use the "cache.app.taggable" adapter or give another tag aware adapter explicitly to the "tags" option. Using the "cache.app" adapter with the "tags" option will throw an exception in Symfony 8.0.');
```

Comment:
let's add the code throwing the exception in a comment

Reply:
Her you go